### DISCUSSION TITLE
error  mongoose

### DISCUSSION THREAD

**Discussion by antonellocaroli** [jcorporation/myMPD#1198] [Q&A] [answered]
myMPD version: 13.0.6

**Describe the bug**

when I use a particular linux system loaded in ram, I get this error:

`/usr/bin/mympd -s

07:22:45 NOTICE   mympd     Cache dir: "/var/cache/mympd"
07:22:45 NOTICE   mympd     Work dir: "/var/lib/mympd"
07:22:45 NOTICE   mympd     Config dir: "/var/lib/mympd/config"
07:22:45 NOTICE   mympd     Lua is disabled, ignoring lua settings
07:22:45 NOTICE   mympd     Setting loglevel to NOTICE
5f8a0  1 mongoose.c:6500:mg_open_listen socket: 97
5f8a0  1 mongoose.c:4257:mg_listen      Failed: http://[::]:80, errno 97
Exiting erroneous, thank you for using myMPD`

when the system is not loaded in ram it works normally.
Can you tell where the error comes from? Maybe I am missing something when loading the system in ram.

**Comment by jcorporation** [comment, marked as answer]
Is IPv6 enabled? Enable it or change the listening host to 0.0.0.0.
